src/modules/catalog.js

```diff
@@ -888,7 +895,7 @@ class Catalog {
 
     return fetch(requestUrl, {
       method: 'POST',
-      body: JSON.stringify(parameters),
+      body: JSON.stringify(toSnakeCaseKeys(params)),
```

Comment:
Do we need to add backwards compatibility for parent_id here?
Also is there a way to limit this to the first level? If we add support for `data` object under `itemGroups` we don't want to convert keys under that. The person implementing it might miss this. Can we either limit the conversion to the root level here or add a comment for future?

Reply:
toSnakeCaseKeys second parameter is recurse, it defaults to false so I think that keeps it at the first level. Is that what you mean?